packages/py-moose-lib/moose_lib/dmv2/olap_dictionary.py

```
@@ -0,0 +1,547 @@
+        elif config.source_tables:
+            self.source_tables = [
+                _format_table_reference(t) for t in config.source_tables
+            ]
+        else:
+            self.source_tables = []
+
+        if name in _olap_dictionaries:
+            raise ValueError(f"OlapDictionary '{name}' is already registered")
+        _olap_dictionaries[name] = self
+
+    def get(self, attr: str, *keys) -> str:
```

Ruff (0.15.7)

[warning] Missing type annotation for `*keys`

(ANN002)